src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  sourceString.replace(/;\s*;/g, ' ').trim();
+
+  const styleRules = sourceString.split(';');
+
+  const result = {};
+
+  styleRules.forEach((element) => {
+    const [key, value] = element.split(':', 2);
```

Comment:
[CODE KNOWLEDGE] - There is no need to limit the `split` operation to 2 parts. If a property value contains a colon, it will be incorrectly split. Omit the second argument to allow for any number of colons within the property value.